packages/auth/src/authClients/netlify.ts

```diff
@@ -8,14 +8,17 @@ export const netlify = (client: NetlifyIdentity): AuthClient => {
   return {
     type: 'netlify',
     client,
-    login: () => {
+    login: (redirectUrl) => {
       return new Promise((resolve, reject) => {
         let autoClosedModal = false
         client.open('login')
         client.on('login', (user) => {
           // This closes the modal which pops-up immediately after you login.
           autoClosedModal = true
           client.close()
+          if (typeof redirectUrl != "undefined" && redirectUrl != null) {
+            window.location.href = redirectUrl;
+          }
```

Comment:
Does redirecting prevent the user getting resolved and then having the user metadata stored?

Reply:
@dthyresson @Tobbe Tested on my end as well, this is working great and doesn't require any changes to netlify.ts.  Appreciate the feedback and quick response.  Only a few days into testing Redwood and this is super helpful.